mitmproxy/addons/export.py

```diff
@@ -39,7 +39,8 @@ def cleanup_response(f: flow.Flow) -> http.Response:
         raise exceptions.CommandError("Can't export flow with no response.")
     assert isinstance(f, http.HTTPFlow)
     response = f.response.copy()  # type: ignore
-    response.decode(strict=False)
+    if response.content:
+        response.decode(strict=False)
```

Comment:
Should we rather fix `.decode()` to not update .content if .raw_content is empty?

Reply:
That's what I thought too, but it seems `raw_content` being `None` and `b""` have different semantics and I'm not sure what I'm missing by treating both the same.

Reply:
See https://github.com/mitmproxy/mitmproxy/pull/7702. We can just check for truthiness in this case.